staging/src/k8s.io/cli-runtime/pkg/printers/name.go

```diff
@@ -38,6 +38,9 @@ type NamePrinter struct {
 	// took place on an object, to be included in the
 	// finalized "successful" message.
 	Operation string
+	// WithNamespace indicates whether a namespace should be
+	// printed along side the "resource/name" pair for an object.
+	WithNamespace bool
```

Comment:
when would `WithNamespace` be true?

Reply:
Added a description about when to be enabled ([`4d46f00` (#94978)](https://github.com/kubernetes/kubernetes/pull/94978/commits/4d46f00b701309f920ff8fefd659a0a363a7cbba)).